src/sentry/models/activity.py

```diff
@@ -33,20 +34,17 @@
 
 class ActivityManager(BaseManager["Activity"]):
     def get_activities_for_group(self, group: Group, num: int) -> Sequence[Activity]:
-        from sentry.issues.priority import PRIORITY_LEVEL_TO_STR
-
         activities = []
         activity_qs = self.filter(group=group).order_by("-datetime")
         initial_priority = None
 
         if not features.has("projects:issue-priority", group.project):
             activity_qs = activity_qs.exclude(type=ActivityType.SET_PRIORITY.value)
         else:
-            event_metadata = group.get_event_metadata()
             # Check if 'initial_priority' is available and the feature flag is on
-            initial_priority_key = event_metadata.get("initial_priority")
+            initial_priority_key = group.get_event_metadata().get("initial_priority")
```

Comment:
nit: this isn't a key for a mapping anymore, so maybe just name it `initial_priority_value`